docs/user/custom_worker.md

```diff
@@ -0,0 +1,149 @@
+# Change how your algorithm samples (Implement a Custom Worker)
+
+```eval_rst
+In sampling, :code:`Worker` perform rollouts to get samples. In this tutorial,
+we will implement a custom worker to meet the requirements of the RL2
+algorithm :cite:`duan2016rl`.
```

Comment:
Explain these special requirements here, so that people can understand what you are trying to achieve in this file BEFORE they read the rest.